bin/docker-entrypoint

```diff
@@ -1,5 +1,23 @@
 #!/bin/bash -e
+set -v
 
-# Add any container initialization steps here
+# Only if we are in an app VM, not a release command
+if [ -z "$RELEASE_COMMAND" ]; then
+  # move the database url so Rails won't use it
+  if [ -n "$DATABASE_URL" ]; then
+    export PG_DATABASE_URL="$DATABASE_URL"
+    unset DATABASE_URL
+  fi
+
+  if [ -n "$FORCE_DB_IMPORT" ]; then
+    rm -f storage/production.sqlite3
```

Comment:
Scary! Should this also be conditional on `PG_DATABASE_URL` being set? To something reasonable?

I suppose ideally a force import would run `bin/sequel` first and only then clobber the existing thing on success?

Reply:
When testing on staging, I kept getting stuck with a file that existed but didn’t have the full schema or the full db import from postgres… so deleting the sqlite file and recreating it from postgres seemed like the only thing that would work for sure. I guess the good news is that I already merged this, it worked, and then I immediately merged the next PR that removed the env var and shrank the VM size.